maestro-ios-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -168,8 +165,8 @@ class XCTestDriverClient(
         return executeJsonRequestUNCHECKED("eraseText", EraseTextRequest(charactersToErase, appIds))
     }
 
-    fun deviceInfo(): Response {
-        return executeJsonRequestUNCHECKED("deviceInfo", Unit)
+    fun deviceInfo(httpUrl: HttpUrl = client.xctestAPIBuilder("deviceInfo").build()): Response {
```

Comment:
why this changed?

Reply:
To make this function testable, so that we can configure HTTP URL and test with mockwebserver